hack/generate_clients.sh

```diff
@@ -4,7 +4,6 @@ set -o nounset
 set -o pipefail
 
 go install k8s.io/code-generator/cmd/client-gen@latest
-go get kubevirt.io/api
```

Comment:
why is this removed?

Reply:
This is not needed because kv api is already in go.mod

But more than that, after the release of kv v1.2.0, this line upgrades kv api package, and the git diff fails later.